.github/workflows/ci.yml

```diff
@@ -339,6 +283,12 @@ jobs:
         key: unstructured-ingest-${{ runner.os }}-${{ matrix.python-version }}-${{ env.GHA_CACHE_KEY_VERSION }}-${{ hashFiles('requirements/ingest/*.txt') }}
         # Due to the dependency on setup_ingest, the cache should exist before this is ran. Set failed if it wasn't found.
         fail-on-cache-miss: true
+    - name: Setup virtual environment (no cache hit)
+      if: steps.virtualenv-cache.outputs.cache-hit != 'true'
+      uses: ./.github/actions/create-ingest-cache
+      with:
+        python-version: ${{ matrix.python-version }}
+        force-recreate: 'true'
```

Comment:
if we fail on cache miss I would think that means we never get to the next step. is that not true??

Reply:
...at which point checking cache-hit seems moot